pkg/kv/kvprober/kvprober.go

```diff
@@ -367,6 +389,12 @@ func (p *Prober) readProbeImpl(ctx context.Context, ops proberOpsI, txns proberT
 	}
 
 	p.metrics.ProbePlanAttempts.Inc(1)
+	var finishAndGetRecording = func() tracingpb.Recording { return nil }
+
+	// trace kv prober requests if tracing is enabled
```

Comment:
I think we can remove this comment. It doesn't say anything that is not obvious from reading the code.

Reply:
Yes, removed now.